### PR TITLE
Us 47

### PR DESCRIPTION
**Pull request by ajcallos**
Fixed system tray icon merge conflict.

### CODE REVIEW

**Comment by cghende1** [comment]
It is working well on Mac.

File: src/net/sf/memoranda/ui/App.java [hunk 1]
```diff
@@ -98,6 +104,51 @@ else if (
 		}
 		if (!Configuration.get("SHOW_SPLASH").equals("no"))
 			splash.dispose();
+		
+		if (SystemTray.isSupported()){
+			try {
+				initTray();
+			} catch (AWTException ex){
+				//TODO:
```
Comment (cghende1) [inline src/net/sf/memoranda/ui/App.java]: We just gonna swallow this?

File: src/net/sf/memoranda/ui/App.java [hunk 2]
```diff
@@ -98,6 +104,51 @@ else if (
...
+	}
+	
+	/** 
+	 * Initializes the Memoranda icon in the system tray
+	 * Uses the oracle example at https://docs.oracle.com/javase/tutorial/uiswing/misc/systemtray.html as the baseline
+	 * @author ajcallos 
+	 * added 2/16/2016 
+	 */
+	private void initTray() throws AWTException	{
+		final String iconPath = "resources/memorandaTrayIcon.gif";
```
Comment (cghende1) [inline src/net/sf/memoranda/ui/App.java]: This should be a constant in the class.

**Comment by jgerszty** [comment]
Not surprisingly, does not work on Gnome. But that's okay because Gnome support is not a feature. +1

**Comment by cmbartee** [comment]
+1 system tray icon working

**Pull request merged**